src/country.ts

```diff
@@ -568,42 +656,52 @@ export const allCountries: Country[] = [
   { name: 'Seychelles', code: 'SC' },
   { name: 'Sierra Leone', code: 'SL' },
   { name: 'Singapore', code: 'SG' },
+  { name: 'Sint Maarten (Dutch part)', code: 'SX' },
   { name: 'Slovakia', code: 'SK' },
   { name: 'Slovenia', code: 'SI' },
   { name: 'Solomon Islands', code: 'SB' },
   { name: 'Somalia', code: 'SO' },
   { name: 'South Africa', code: 'ZA' },
+  { name: 'South Georgia and the South Sandwich Islands', code: 'GS' },
+  { name: 'South Sudan', code: 'SS' },
   { name: 'Spain', code: 'ES' },
   { name: 'Sri Lanka', code: 'LK' },
   { name: 'Sudan', code: 'SD' },
-  { name: 'South Sudan, Republic of', code: 'SS' },
   { name: 'Suriname', code: 'SR' },
-  { name: 'Eswatini', code: 'SZ' },
+  { name: 'Svalbard and Jan Mayen', code: 'SJ' },
+  { name: 'Swaziland', code: 'SZ' },
   { name: 'Sweden', code: 'SE' },
   { name: 'Switzerland', code: 'CH' },
   { name: 'Syrian Arab Republic', code: 'SY' },
-  { name: 'Taiwan', code: 'TW' },
+  { name: 'Taiwan, Province of China', code: 'TW' },
```

Comment:
Leave it as Taiwan for now 😅

Reply:
That's exactly what I thought 😅, geopolitics are everywhere hahaha